src/artifact_providers/base.ts

```diff
@@ -61,14 +62,47 @@ export interface LocalArtifact extends AbstractArtifact {
 }
 
 /**
- * Fitlering options for artifacts
+ * Raw filtering options for artifacts
  */
-export interface FilterOptions {
+export interface RawFilterOptions {
```

Comment:
Please add a note that the raw filter options should be processed by `parseFilterOptions`.